### PR TITLE
tewisay: unstable-2017-04-14 -> unstable-2022-11-04

### PR DESCRIPTION
**Pull request by Madouura**
###### Description of changes
Version bump.
Also old repo was deleted, arch uses this new one so we'll use it as well.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Madouura** [comment]
Result of `nixpkgs-review pr 207625` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tewisay</li>
  </ul>
</details>

**Comment by Madouura** [comment]
cc @SuperSandro2000 for review/merge

File: pkgs/tools/misc/tewisay/default.nix [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
-    sha256 = "1na3xi4z90v8qydcvd3454ia9jg7qhinciy6kvgyz61q837cw5dk";
+    rev = "caa5b0131dda868f656716d2107f02d04d1048d4";
+    hash = "sha256-E492d8P/Bek9xZlJP+k9xvIJEFtA1YrIB/pogvz3wM4=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  vendorHash = "sha256-WcpRJ31kqWA255zfjuWDj0honJgSGdm4ONx2yOKk7/g=";
 
-  goDeps = ./deps.nix;
+  # Currently hard-coded, will be fixed by developer
```
Comment (Mindavi) [inline pkgs/tools/misc/tewisay/default.nix:21]: Where was this communicated?

Comment (Madouura) [reply pkgs/tools/misc/tewisay/default.nix:21]: https://github.com/raymond-w-ko/tewisay/blob/caa5b0131dda868f656716d2107f02d04d1048d4/main.go#L138
I've been making a habit of searching for ``/usr`` and other hard-codes recently.

Comment (SuperSandro2000) [reply pkgs/tools/misc/tewisay/default.nix:21]: Would be nice to link those  in the comment.

**Pull request merged**